client/my-sites/plans-features-main/components/free-free-dialog.tsx

```diff
@@ -0,0 +1,290 @@
+}
+
+export function FreeFreeDialog( {
+	freeSubdomain,
+	onFreePlanSelected,
+	onPlanSelected,
+	onClose,
+}: {
+	freeSubdomain: string;
+	onClose: () => void;
+	onFreePlanSelected: () => void;
+	onPlanSelected: () => void;
+} ) {
+	const translate = useTranslate();
+	const currencyCode = useSelector( getCurrentUserCurrencyCode ) ?? 'USD';
+
+	const planConstantObj = applyTestFiltersToPlansList( PLAN_PERSONAL, undefined );
+	const planProductId = planConstantObj.getProductId();
+	const rawPrice = useSelector( ( state ) => getPlanRawPrice( state, planProductId, true ) );
+
+	return (
+		<Dialog
+			isBackdropVisible={ true }
+			isVisible={ true }
+			onClose={ onClose }
+			showCloseIcon={ true }
+		>
+			<Global
+				styles={ css`
+					.dialog__backdrop.is-full-screen {
+						background-color: rgba( 0, 0, 0, 0.6 );
+					}
+					.ReactModal__Content--after-open.dialog.card {
+						border-radius: 4px;
+						width: 605px;
+					}
+				` }
+			/>
+			<DialogContainer>
+				<Heading>{ translate( "Don't miss out" ) }</Heading>
+				<TextBox>
+					{ translate( "With a Free plan, you'll miss out on a lot of great features:" ) }
+				</TextBox>
+				<List>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No free custom domain:{{/strong}} Your site will be shown to visitors as {{strong}}%(freeSubdomain)s{{/strong}}',
+								{
+									args: {
+										freeSubdomain,
+									},
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No ad-free experience:{{/strong}} Your visitors will see external ads on your site.',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No unlimited professional customer support{{/strong}} (only community forums)',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No extra storage:{{/strong}} You only get 1GB for photos, videos, media, and documents.',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+				</List>
+				<TextBox>
+					{ rawPrice &&
+						translate(
+							'Unlock {{strong}}all of{{/strong}} these features with a Personal plan,starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```

Comment:
```suggestion
							'Unlock {{strong}}all of{{/strong}} these features with a Personal plan, starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
```
Missing space after comma :)

Reply:
The `Personal` string should come from the plan object. You can use `getPlan( <slug> )?.getTitle()` to get this. If it's directly passed to `translate`, it might get literally translated.

Reply:
I think an easy way to avoid similar issues in the modal can be to pass a `suggestedPlanSlug` to this modal. This way, we'll be forced to fetch the title from the plan object and will reduce the chances of hard-coded stuff getting through. 🙂

Reply:
Thanks @aneeshd16 this is a good suggestion!

> I think an easy way to avoid similar issues in the modal can be to pass a suggestedPlanSlug to this modal